server/proxy/proxy.go

```diff
@@ -225,7 +228,28 @@ func (p *Proxy) NotifyMessageHasBeenPosted(post *model.Post, cc *apps.Context) e
 
 	subs := []*apps.Subscription{}
 	subs = append(subs, postSubs...)
-	mentions := model.PossibleAtMentions(post.Message)
+
+	// This code is copied over from mattermost-server/app.possibleAtMentions
+	possibleAtMentions := func(message string) []string {
```

Comment:
Can we define this function somewhere else? I want to keep these `Notify*` functions as simple as possible

Reply:
Sure